extensions/image-preview/media/main.js

```diff
@@ -152,36 +173,53 @@
 		}
 	});
 
-	container.addEventListener('click', (/** @type {MouseEvent} */ e) => {
+	container.addEventListener('mousedown', (/** @type {MouseEvent} */ e) => {
```

Comment:
Also do we need both mousedown and `click`? I think we should be able to handle everything in `click`?

Reply:
```canClickToZoom``` and the ```'mousedown'``` event handler where added because the ```'click'``` event is fired when mouse button is lifted, causing a zoom on first click if the user does not release the mouse button before ```isActive``` is set to true. These changes ensures that only clicks that are initiated after focus is acquired results in zooming.

I think this change is good because the outcome of the first click will seem inconsistent otherwise. I will try to get the same effect without the use of ```canClickToZoom``` and the ```'mousedown'``` event handler, but I'm not sure I will succeed.